### DISCUSSION TITLE
[firebase_dynamic_links] Web

### DISCUSSION THREAD

**Discussion by felixwortmann** [firebase/flutterfire#4431] [Feature request]
I think it would be a really useful feature to make it possible to handle dynamic links in web. 
Obviously you want to redirect the user to the store page if the app is not installed. 

If you use it on desktop though, you might want to handle the dynamic link in the browser version of your app.

I know that there is a "Deeplink-URL" parameter in Firebase where you can enter a default url for desktop but it is not possible to pass any of the dynamic link parameters as far as I know.

Also I am not sure if this is even possible and supported by Firebase.
It would be a very useful feature for a lot of developers imo.

**Comment by frankuserconnect** [comment]
@felixwortmann 

Hi! I am also having the same issue. Did you figure it out.

**Reply by felixwortmann** [reply]
Unfortunately no, I just passed my params by encoding them into the url

**Comment by Vaytrex** [comment]
For anyone landing here after the Aug 2025 shutdown: there's no drop-in. You need a URL that branches on User-Agent (App Store / intent:// with browser_fallback_url / website) and skips redirecting crawlers so previews survive. Full write-up with the intent format: https://linkbah.app/firebase-dynamic-links-alternative
